app/lib/import_export.rb

```diff
@@ -269,7 +269,7 @@ def extract_patients_details(patients, fields)
         patient_details[:end_of_monitoring] = patient.end_of_monitoring if fields.include?(:end_of_monitoring)
         patient_details[:expected_purge_ts] = patient.expected_purge_date_exp if fields.include?(:expected_purge_ts)
         patient_details[:full_status] = patient.status&.to_s&.humanize&.downcase if fields.include?(:full_status)
-        patient_details[:status] = patient.status&.to_s&.humanize&.downcase&.sub('exposure ', '')&.sub('isolation ', '') if fields.include?(:status)
+        patient_details[:status] =  patient.status_as_string if fields.include?(:status)
```

Comment:
Extraneous space
```suggestion
        patient_details[:status] = patient.status_as_string if fields.include?(:status)
```

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/1016/commits/3881a715e296d7607ca4ae30ad7a13805b9c569c.